recipes/doxdox/recipe.yaml

```diff
@@ -0,0 +1,39 @@
+context:
+  version: 4.0.0-preview.24
```

Comment:
This is a pre-release, right? We would prefer to have releases only.

Reply:
Yes, that version seems to be a prerelease.  I was able to figure out what the latest stable version and have switched the recipe to that instead - see the latest commit for the change.